client/my-sites/email/form/mailboxes/components/utilities/mailbox-operations.ts

```diff
@@ -12,8 +13,22 @@ export class MailboxOperations {
 	}
 
 	validateLocal() {
```

Comment:
This method was a little bit hard to decipher IMO. Maybe something like this could improve the readability?

```ts
validateLocal() {
    this.mailboxes.forEach( ( mailbox ) => {
        const otherMailboxNames = this.mailboxes
            .filter( ( m ) => m.getFieldValue( FIELD_UUID ) !== mailbox.getFieldValue( FIELD_UUID ) )
            .map( ( mailbox ) => mailbox.getFieldValue< string >( FIELD_MAILBOX ) );

        mailbox.validate(
            true,
            mailbox.getPreviouslySpecifiedMailboxNameValidators( otherMailboxNames as string[] )
        );
    } );
}
```

Reply:
You've got a point. This is much simpler.